### PR TITLE
Quick start for monitoring. 

### PR DESCRIPTION
**Pull request by saketbairoliya2**
Here is the mapping JIRA task: https://truera.atlassian.net/browse/MUX-1513

### CODE REVIEW

File: quickstart/monitoring/monitoring-quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,375 @@
+{
```
Comment (joshreini1) [inline quickstart/monitoring/monitoring-quickstart.ipynb:1]: <p>do we need ignore_version_mismatch flag?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/15/discussion/#comment-1321852858'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (joshreini1) [inline quickstart/monitoring/monitoring-quickstart.ipynb:1]: <p>Is record-level custom metrics available to include here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/15/discussion/#comment-1321852861'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (saketbairoliya2) [reply quickstart/monitoring/monitoring-quickstart.ipynb:1]: Yeah - I can add that.

Comment (saketbairoliya2) [reply quickstart/monitoring/monitoring-quickstart.ipynb:1]: No, this is purposefully removed from QuickStart docs as we wanted to keep this simple.

**Comment by joshreini1** [comment]
@saketbairoliya2 can you PR this into release/rc-1.39 instead? That branch is now available and will become release/prod later today

**Comment by saketbairoliya2** [comment]
> @saketbairoliya2 can you PR this into release/rc-1.39 instead? That branch is now available and will become release/prod later today

- Updated the base branch. @joshreini1

**Pull request merged**